evm/src/NttManager/NttManager.sol

```diff
@@ -386,6 +391,28 @@ contract NttManager is INttManager, RateLimiter, ManagerBase {
         );
     }
 
+    function _validateTransfer(
+        uint16 recipientChain,
+        bytes memory transceiverInstructions
+    ) internal view {
+        if (_getPeersStorage()[recipientChain].peerAddress == bytes32(0)) {
```

Comment:
Another sanity check against https://github.com/wormhole-foundation/example-native-token-transfers/pull/364 (Wormhole Chain ID is canonical). This will access the correct spot, right?

Looks good otherwise